frontend/src/Components/Karma.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useEffect, useState} from 'react';
+import './KarmaCalculator.scss';
+export function Karma() {
+    const [allPostsValue, setP] = useState(0); // sum of votes for all posts
+    const [allCommentsValue, setC] = useState(0); // sum of votes for all comments
+    const [profileVotesCount, setUCount] = useState(0); //count of votes in profile
+    const [profileVotingResult, setU] = useState(0); // sum of votes in profile
+    const [punishment, setPunishment] = useState(0); // penalty set by moderator
+
+    useEffect(() => {
+         setU( clamp(profileVotingResult, -profileVotesCount, profileVotesCount));
+    },[profileVotesCount]);
```

Comment:
Linter is correct, `profileVotingResult` should be also present in the effect dependency list.

Reply:
done